content/docs/introduction/comparison.md

```diff
@@ -132,14 +132,13 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
```

Comment:
We can remove the "currently" and remove the link, as it looks like there won't be one.